Dockerfile

```diff
@@ -14,7 +14,7 @@ COPY main.go main.go
 COPY controllers/ controllers/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=$TARGETARCH GO111MODULE=on go build -a -o manager main.go
```

Comment:
Is `TARGETARCH` a built-in? I don't see it mentioned in the documentation.

Reply:
https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope